types/prompts.ts

```diff
@@ -27,9 +34,9 @@ export type PromptConfig<T extends GenericPromptResponse> = {
   when?: PromptWhen;
   pageSize?: number;
   default?: PromptOperand | ((answers: T) => PromptOperand);
-  validate?:
-    | ((answer?: string) => PromptOperand | Promise<PromptOperand>)
-    | ((answer: string[]) => PromptOperand | Promise<PromptOperand>);
+  transformer?: (input: string) => string | undefined;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  validate?: (answer?: unknown | any) => PromptOperand | Promise<PromptOperand>;
```

Comment:
If `unknown` doesn't work, you can just use `any`. Typing something as  `Type | any` is the same as just typing it as `any`

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1293/commits/ad7654ea3612b8f2294af6d5063d4e2360a2c5ec